docs/content/docs/recipes/signals.md

```diff
@@ -0,0 +1,516 @@
+draft: true
+---
+
+In this recipe, we will go over the `PresenceTracker` example to learn how the `Signaler` DataObject is used in a Fluid application to deal with a couple user presence scenarios. The two forms of presence covered in this example are focus tracking and mouse tracking. This article will go over how the `Signaler` DataObject is used in both the `MouseTracker` and `FocusTracker` classes to achieve user presence in a Fluid application. You can find the completed code for this example in the Fluid Repo [here](https://github.com/microsoft/FluidFramework/tree/main/examples/data-objects/presence-tracker).
+
+## Creation
+
+The `Signaler` DataObject is first included in the `initialObjects` of the Fluid `ContainerSchema` which defines which shared objects will be at our disposal in the new container.
+
+```typescript
+const containerSchema: ContainerSchema = {
+    initialObjects: {
+        /* [id]: DataObject */
+        signaler: Signaler,
+    },
+};
+```
+
+The `FluidContainer` is then created (or fetched if it already exists) alongside the `services` object, since audience will be required in this application. To learn more about how to create a new Fluid container and load an existing container, see our [DiceRoller Tutorial](https://fluidframework.com/docs/start/tutorial/).
+
+## MouseTracker
+
+Let's look at how the `MouseTracker` class uses the `Signaler` DataObject to achieve the first form of user presence within the application.
+
+The class defines the `mouseSignalType` that will be sent and listened to the connected clients when there is a presence change:
+
+```typescript
+export interface IMouseTrackerEvents extends IEvent {
+    (event: "mousePositionChanged", listener: () => void): void;
+}
+
+export class MouseTracker extends TypedEventEmitter<IMouseTrackerEvents> {
+    private static readonly mouseSignalType = "positionChanged";
+
+    /*...*/
+}
+```
+The class also initializes a local map of `IMousePosition` values for all of the connected clients. Each userID can have multiple clientIDs (e.g. same user on separate devices), which explains the nested `Map`. This local map is what populates the view and what will be updated on `mouseSignalType` signals:
+
+```typescript
+export interface IMouseTrackerEvents extends IEvent {
+    (event: "mousePositionChanged", listener: () => void): void;
+}
+
+export interface IMousePosition {
+    x: number;
+    y: number;
+}
+
+export class MouseTracker extends TypedEventEmitter<IMouseTrackerEvents> {
+    private static readonly mouseSignalType = "positionChanged";
+
+    /**
+     * Local map of mouse position status for clients
+     *
+     * ```
+     * Map<userId, Map<clientid, position>>
+     * ```
+     */
+    private readonly posMap = new Map<string, Map<string, IMousePosition>>();
+
+    /*...*/
+}
+```
+Now that we have clarity about what `posMap` and `mouseSignalType` are, we can move to the constructor to see what other behavior our `MouseTracker` has.
```

Comment:
I like that you lead with "here's the core data that the tracker is holding (posMap)".  To segue from the data to the constructor requirements, maybe something like

```suggestion
Now that we have clarity about what `posMap` and `mouseSignalType` are, we need to use the `audience` and `Signaler` to send, receive, and interpret the positions.  We provide these in the constructor.
```